web/templates/settings/uploaded_files_list.hbs

```diff
@@ -20,13 +20,13 @@
     <td class="upload-size" >{{ size_str }}</td>
     <td class="actions">
         <span class="edit-attachment-buttons">
-            <a type="submit" href="/user_uploads/{{path_id}}" class="btn no-style tippy-zulip-delayed-tooltip" data-tippy-content="{{t 'Download' }}" id="download_attachment" download>
```

Comment:
Let's use the `button` tag over here instead, that would also help us remove the unwanted underlines on the icon on hover (see the tiny "dot like thing" in image below).

![download_btn_underline](https://github.com/zulip/zulip/assets/82862779/2df90403-6899-4837-a695-5a0db01f9eec)